inference-engine/tests/functional/plugin/gpu/shared_tests_instances/plugin_config.cpp

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "functional_test_utils/plugin_config.hpp"
+
+void ConfigurePlugin(LayerTestsUtils::LayerTestsCommon* test) {
```

Comment:
Looks like, we should remove this function. Maybe `PreparePluginConfiguration()` I don't have better ideas :)

Reply:
@iefode I guess you meant 'rename' but not 'remove'? :) If so, I think `ConfigurePlugin` is a short and pretty meaningful name

Reply:
Yes, it was typo. But it can mislead.. I think it will be be good for `plugin->SetConfig()`. But it is just my opinion. In general, changes LGTM besides this name